build-system/pr-check/performance-tests.js

```diff
@@ -19,13 +19,13 @@
  * @fileoverview Script that runs the performance tests during CI.
  */
 
-const {downloadNomoduleOutput, timedExecOrDie} = require('./utils');
+const {fetchBuildOutput, timedExecOrDie} = require('./utils');
 const {runCiJob} = require('./ci-job');
 
 const jobName = 'performance-tests.js';
 
 function pushBuildWorkflow() {
-  downloadNomoduleOutput(jobName);
+  fetchBuildOutput(jobName);
```

Comment:
> …let's be conservative about edits to `.circleci/config.yml` in particular because (as I've been harping about), each edit to that file will force all in-flight PRs to rebase their branches

We can consolidate merging all of these PRs of mine close together sometime next week, this way they're still separate PRs (which can be individually rolled back if needed) and the interruption should be minimal

Reply:
Done